doc/pre-render-scripts.md

```diff
@@ -0,0 +1,48 @@
+# Pre-render scripts
+
+The directory __bosh/releases/pre_render_scripts__ contains scripts
+added to the `quarks` property.
+
+The structure of this directory is expected to be
+`<instance_group>/<job>/<type>/<script>`, where `<type>` is one of
+
+  - `jobs`
+  - `bpm`
+  - `ig_resolver`
+
+This type details the exact location of where the patch executes.
+
+## Background
+
+[Pre render scripts] are kubecf's equivalent of its predecessors' (SCF
+v2) [patches] scripts. They, like them, enable developers and
+maintainers to apply general patches to the sources of a job
+(i.e. configuration templates, script sources, etc.) before that job
```

Comment:
Should this be `e.g.`?

Reply:
Fixed with commit 9918b20dcb257d8865a289cee9653b710a4662a0.